components/ingest-service/pipeline/processor/validator_run.go

```diff
@@ -0,0 +1,37 @@
+package processor
```

Comment:
If the original message is missing the "enitiy_uuid" or "run_id" the message is removed from the pipeline with an error.